core/db/migrate/20180613080857_rename_guest_token_to_token.rb

```diff
@@ -0,0 +1,5 @@
+class RenameGuestTokenToToken < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :spree_orders, :guest_token, :token
```

Comment:
@kushniryb I think we need to also drop the index and create a new one?

Reply:
There's no need to do that, Rails does it automatically.